FlowCryptTests/Functionallity/WKDURLs/WKDURLsConstructorTests.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  WKDURLsTests.swift
+//  FlowCryptTests
+//
+//  Created by Yevhen Kyivskyi on 17.05.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import XCTest
+
+class WKDURLsTests: XCTestCase {
+    
+    var constructor: WKDURLsConstructorType!
+
+    override func setUp() {
+        constructor = WKDURLsConstructor()
+    }
+
+    override func tearDown() {
+        constructor = nil
+    }
+
```

Comment:
Please use specifically these four test cases mentioned in the issue to ensure results match.

```

    recipient.hello@example.com DIRECT -> https://example.com/.well-known/openpgpkey/hu/1sbjrcaf8m3zckmmuej93nx61yn1sttg?l=recipient.hello
    recipient.hello@example.com ADVANCED -> https://openpgpkey.example.com/.well-known/openpgpkey/example.com/hu/1sbjrcaf8m3zckmmuej93nx61yn1sttg?l=recipient.hello
    UPPER@EXAMPLE.COM DIRECT -> https://example.com/.well-known/openpgpkey/hu/awhcnhf7a4ax8qha5u1rwymkfaswmjz8?l=UPPER
    UPPER@EXAMPLE.COM ADVANCED -> https://openpgpkey.example.com/.well-known/openpgpkey/example.com/hu/awhcnhf7a4ax8qha5u1rwymkfaswmjz8?l=UPPER
```

Reply:
I've tried to encrypt with sha1 and encode with z-base32(and vice-versa) using online services and couldn't get `1sbjrcaf8m3zckmmuej93nx61yn1sttg`. Could you specify the flow, please?

Reply:
If it doesn't give exactly the same string, then it's not useful. The flow is this:

```
const userBytes = user.toLowerCase().utfToBytes();
const hash = sha1(userBytes); // accepts bytes, returns bytes
const encoded = encodeZBase32(hash);
```

If you encode sha1 online, it gives you a hex string, not raw bytes. If you fed that into zbase32 then you'd get a wrong output. You need to feed it output bytes from sha1 directly.

Reply:
Got it, I followed the wrong flow, now it works. Updating PR. Thank you!

Reply:
done